bindings.c

```diff
@@ -3794,6 +3799,90 @@ static uint64_t get_reaper_age(pid_t pid)
 	return procage;
 }
 
+/*
+ * Returns 0 on success.
+ * It is the caller's responsibility to free `return_usage`, unless this
+ * function returns an error.
+ */
+static int read_cpuacct_usage_all(char *cg, char *cpuset, struct cpuacct_usage **return_usage)
+{
+
+	int cpucount = get_nprocs();
```

Comment:
`get_nprocs()` always returns a positive value? If not the `malloc()` below will be might funny...

Reply:
Manpage says: *The function get_nprocs() returns the number of processors currently available in the  system.* So I expect it should be always at least `1`.